generatePrefixesList.js

```diff
@@ -0,0 +1,427 @@
+import axios from "axios";
```

Comment:
I'm a bit confused.. Why do we want two generatePrefixesList.js files?

Reply:
You're right, this file was added by mistake and should be removed, it is not used

Reply:
But you were making changes in /generatePrefixesList.js that were not in /src/generatePrefixesList.js... so now you've lost those changes by deleting the file?

Reply:
The changes in  /generatePrefixesList.js are relevant, I was confused why some of my changes where no effect. I was changing the wrong file. The /src/generatePrefixesList.js is the correct and only file being used. That is why the syntax error didn't matter before.

Reply:
ok thanks for clarifying